### PR TITLE
feat: add graph page

### PR DESCRIPTION
**Pull request by Yedidyar**

### CODE REVIEW

File: src/components/navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          <GitHubLogoIcon />
+        </Link>
+      </div>
+    </div>
+  );
+};
+
+export default Navbar;
+
+export function MainNav({
```
Comment (InbarKedem) [inline src/components/navbar.tsx:33]: put into a different file

Comment (InbarKedem) [reply src/components/navbar.tsx:33]: and maybe declare the input types as an interface?

File: src/pages/toasts-history/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  ResponsiveContainer,
+} from "recharts";
+import type { TooltipProps } from "recharts";
+import type {
+  ValueType,
+  NameType,
+} from "recharts/types/component/DefaultTooltipContent";
+import { periodHistory } from "~/data";
+
+const CustomTooltip = ({
```
Comment (InbarKedem) [inline src/pages/toasts-history/index.tsx:18]: `const TooltipContent = ({`

File: src/pages/toasts-history/index.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  active,
+  payload,
+  label,
+}: TooltipProps<ValueType, NameType>) => {
+  if (active) {
```
Comment (InbarKedem) [inline src/pages/toasts-history/index.tsx:23]: `if (active && payload && payload.[0]) {`

File: src/components/criminals.tsx [hunk 4]
```diff
@@ -12,7 +14,10 @@ const criminalsTypes = {
 
 export const Criminals = () => {
   const [open, setOpen] = useState(false);
```
Comment (InbarKedem) [inline src/components/criminals.tsx:16]: `const [isOpen, setIsOpen] = useState(false);`

File: src/components/criminals.tsx [hunk 5]
```diff
@@ -12,7 +14,10 @@ const criminalsTypes = {
...
-  const { data } = api.criminal.getAll.useQuery(undefined, { enabled: open });
+  const { data, isLoading } = api.criminal.getAll.useQuery(undefined, {
```
Comment (InbarKedem) [inline src/components/criminals.tsx:17]: why are you sending undefined?

Comment (Yedidyar) [reply src/components/criminals.tsx:17]: if I wont do it I cannot access to the options

**Pull request merged**